components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml

```diff
@@ -7853,6 +7951,7 @@ paths:
         - OAuth2Security:
             - apim:common_operation_policy_view
             - apim:common_operation_policy_manage
+            - apim:policies_import_export
```

Comment:
As we are not using this endpoint, we don't need this scope addition.

Reply:
Devops has to use this to get the `APIPolicyId` for the deletion. Or else, devops should have the `apim:common_operation_policy_view` scope.

Reply:
Since we are using this in apictl delete we need to keep this scope.